staking-pool/README.md

```diff
@@ -97,7 +102,9 @@ This balance consist of the initial contract balance, and all delegator account
 When the method is called the contract uses the current total account balance (without attached deposit) and the subtracts the last total account balance.
 The difference is the total reward that has to be distributed.
 
-The fraction of the reward is awarded to the contract owner. The fraction is configurable by the owner, but can't exceed 1.
+The fraction of the reward is awarded to the contract owner. The fraction is configurable by the owner, but can't exceed 100%.
+Note, that it might be unfair for the participants of the pool if the owner changes reward fee. But this owner will lose trust of the
+participants and it will lose future revenue in the long term. This should be enough to prevent owner from abusing reward fee.
```

Comment:
It could also be the case that they could change the reward fee to make their pool more attractive.